public/index.html

```diff
@@ -7,6 +7,6 @@
   <title>Document</title>
```

Comment:
- [ ] It would be better if we align the Nav header "Bookstore CMS" with the "BookList" component , also the "user icon" is missing from the design